### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (input[placeholder="Enter\ your\ project\ name\ here"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2332]
#### Issue
`Form elements must have labels` ([label](https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta#editor)
#### Element path
input[placeholder="Enter\ your\ project\ name\ here"]
#### Snippet
`<input type="text" placeholder="Enter your project name here" value="">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

modal that shows up when clicking to download a file:

<img width="736" alt="Screen Shot 2019-08-02 at 2 42 35 PM" src="https://user-images.githubusercontent.com/5615930/62400395-ee733d00-b533-11e9-9254-4b941b937d27.png">

![2019-08-02 14 43 16](https://user-images.githubusercontent.com/5615930/62400431-09de4800-b534-11e9-8a9a-188bd481c859.gif)

**Comment by BhuwanNaswa** [comment]
Labels
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS3.3.2, #Win10-Chrome, #Win10-Edge, #accessibility, #K4W, #closed
